src/Sylius/Behat/Service/Provider/Calendar.php

```diff
@@ -14,7 +14,7 @@
 namespace Sylius\Behat\Service\Provider;
 
 use Sylius\Bundle\ShippingBundle\Provider\DateTimeProvider;
-use Sylius\Component\Promotion\Provider\DateTimeProviderInterface;
+use Sylius\Calendar\Provider\DateTimeProviderInterface;
 
 final class Calendar implements DateTimeProvider, DateTimeProviderInterface
```

Comment:
this service shoud be removed and replaced by `fakeDateProvider` from `sylius/calendar`